src/components/messenger/list/group-details-panel/index.tsx

```diff
@@ -14,34 +14,43 @@ const cn = bemClassName('group-details-panel');
 
 export interface Properties {
   users: Option[];
-
   onBack: () => void;
   onCreate: (data: { name: string; users: Option[]; image: File }) => void;
 }
 
 interface State {
   name: string;
   image: File | null;
+  errors: {
+    name?: string;
+  };
 }
 
 export class GroupDetailsPanel extends React.Component<Properties, State> {
-  state = { name: '', image: null };
+  state = { name: '', image: null, errors: {} };
 
   createGroup = () => {
     this.props.onCreate({ name: this.state.name, users: this.props.users, image: this.state.image });
   };
 
-  nameChanged = (value) => {
-    this.setState({ name: value });
-  };
+  nameChanged = (value) => this.setState({ name: value });
+  onImageChange = (image) => this.setState({ image });
+  back = () => this.props.onBack();
 
-  onImageChange = (image) => {
-    this.setState({ image });
-  };
+  get isValid() {
+    return typeof this.state.name === 'string' && this.state.name.trim().length > 0;
+  }
 
-  back = () => {
-    this.props.onBack();
-  };
+  get nameError() {
+    if (!this.isValid) {
+      return { variant: 'error', text: 'Please enter a group name.' } as any;
```

Comment:
Let's just rename this to `name is required`? So the error message is one-lined?

Reply:
ok